package.json

```diff
@@ -30,20 +30,32 @@
   },
   "devDependencies": {
     "babel-core": "^6.7.7",
+    "babel-eslint": "^7.0.0",
     "babel-polyfill": "^6.5.0",
     "babel-preset-es2015": "^6.5.0",
+    "babel-preset-node5": "^11.1.0",
     "del": "^2.2.0",
+    "eslint": "^3.8.0",
+    "eslint-config-fb-strict": "^15.0.0",
+    "eslint-config-fbjs": "^1.1.1",
+    "eslint-plugin-babel": "^3.3.0",
+    "eslint-plugin-flowtype": "^2.20.0",
+    "eslint-plugin-react": "^6.4.1",
     "gulp": "^3.9.1",
     "gulp-babel": "^6.1.2",
     "gulp-eslint": "^3.0.0",
-    "gulp-jasmine": "^2.2.1",
+    "gulp-newer": "^1.3.0",
     "gulp-watch": "^4.3.5",
-    "jasmine-co": "^1.1.0",
-    "proxyquire": "^1.7.4"
+    "jest": "^16.0.1"
   },
   "license": "ISC",
   "scripts": {
     "test": "gulp lint test",
     "prepublish": "gulp build"
+  },
+  "jest": {
```

Comment:
by default [jest test environment](https://facebook.github.io/jest/docs/configuration.html#testenvironment-string) is set to `jsdom`. If you are ready to give up the window/globals you can set it to `node` and have faster tests.